specifications/move_adapter/README.md

```diff
@@ -202,11 +202,15 @@ order specified here.
 The adapter performs the following steps for any transaction, regardless of
 the payload:
 
-* Check if the signature in the `SignedTransaction` is consistent with the
-public key and the `RawTransaction` content. If not, this check fails with an
-`INVALID_SIGNATURE` status code. Note that comparing the transaction's public
-key against the sender account's authorization key is done separately in Move
-code.
+* Check if the sender and secondary signers' signatures in the `SignedTransaction`
```

Comment:
The addresses of secondary signers and all signers' signatures are in the authenticator field stored inside SignedTransaction. But yes it might make sense to mention RawTransactionWithData here.